quickstarts/jboss-eap7-with-helm.yaml

```
@@ -0,0 +1,216 @@
+  introduction: |-
+    [Red Hat® JBoss® Enterprise Application Platform (EAP)](https://www.redhat.com/en/technologies/jboss-middleware/application-platform) is an application server. It includes everything needed to build, run, deploy, and manage enterprise Java applications in a variety of environments, including on-premise, virtual environments, and in private, public, and hybrid clouds.
+
+    * **Optimized for OpenShift, Cloud and Containers**
+
+    * **Jakarta EE 8 compatibility**
+
+    * **Lightweight, flexible architecture**
+
+    * **Red Hat Portfolio Integration**
+
+    * **More productive developers with DevOps and Agile Development**
+
+    In this quick start, you will deploy and run a [Jakarta EE application](https://github.com/jboss-eap-up-and-running/eap7-getting-started) with JBoss EAP 7 on OpenShift.
```

I think we should avoid using the future tense of "will" here. It might be better to say something like:
```suggestion
    With this quick start, you can deploy and run a [Jakarta EE application](https://github.com/jboss-eap-up-and-running/eap7-getting-started) with JBoss EAP 7 on OpenShift.
```